lib/errors.js

```diff
@@ -1,39 +1,73 @@
+'use strict';
+
+var util = require('util');
+
+/**
+ * The base class for all thinky errors
+ *
+ * @constructor
+ * @alias Error
+ */
+var ThinkyError = function(message) {
```

Comment:
oh I thought he was asking about why it was being assigned to the variable. Is there a benefit to the alternative approach (seems there's always more for me to learn about javascript :smile:)